repack.go

```diff
@@ -78,9 +78,8 @@ func Repack(engineExt casext.Engine, tagName string, bundlePath string, meta Met
 		"ndiff": len(diffs),
 	}).Debugf("umoci: checked mtree spec")
 
-	diffs = mtreefilter.FilterDeltas(diffs,
-		mtreefilter.MaskFilter(maskedPaths),
-		mtreefilter.SimplifyFilter(diffs))
+	allFilters := append([]mtreefilter.FilterFunc{mtreefilter.SimplifyFilter(diffs)}, filters...)
```

Comment:
Did you mean to put `SimplifyFilter` at the end as it was originally? In which case it should be

```
filters = append(filters, mtree.SimplifyFilter(diffs))`
```

Reply:
No, I meant to put it at the beginning; if it's the first filter run, I guess there will be less overall filtering needed? I can move it to the end if you want, though.

Reply:
The filters will be run the same number of times (each filter is run for each element -- maybe we shouldn't do it that way but :man_shrugging:), but I don't really mind either way. I just think

```
filters = append(filters, mtree.SimplifyFilter(diffs))
```

is slightly less ugly.

Reply:
Oh, ok. I'll switch it then, I figured it chained them all. Agreed it's less ugly :)